rollup-tests.config.js

```diff
@@ -15,6 +16,14 @@ export default {
   },
   treeshake: false, // Disabled for build performance
   plugins: [
+    alias({
+      entries: [
+        {
+          find: '../../build/settings.json',
+          replacement: '../../tests/settings.json',
```

Comment:
In tests the `settings.json` import is re-pointed at a static `settings.json` file in the `tests/` dir.